invenio_theme/assets/semantic-ui/js/invenio_theme/truncate.js

```diff
@@ -0,0 +1,24 @@
+import $ from "jquery";
+
+$(".truncate-button").on("click", function () {
+  const $clickButton = $(this);
+  const siblings = $clickButton.siblings();
+  siblings.first().toggleClass("truncate-2");
+  $clickButton.toggle();
+  siblings.eq(1).toggle();
+});
+
+const ps = document.querySelectorAll(".truncate-container .truncate-text");
```

Comment:
naming police: please give this variable more meaningful name. what does ps mean?